### PR TITLE
JT-26 Spring Security config

### PR DESCRIPTION
**Pull request by freesky15**

### CODE REVIEW

**Comment by saffchen** [comment]
Please order the task number

File: stock/src/main/java/saffchen/entities/PersonEntity.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+@NoArgsConstructor
+@EqualsAndHashCode
+@Table(name = "Person")
+public class PersonEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false, unique = true)
+    private Long id;
+
+    //@Enumerated(value=EnumType.STRING)
```
Comment (saffchen) [inline stock/src/main/java/saffchen/entities/PersonEntity.java:20]: If we don't need this annotation then delete it

File: stock/src/main/java/saffchen/entities/SatelliteEntity.java [hunk 2]
```diff
@@ -15,9 +15,7 @@
 @Table(name = "Satellite")
 public class SatelliteEntity {
     @Id
-    @SequenceGenerator(name = "satellite_id_sequence",
-            sequenceName = "satellite_id_seq")
-    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "satellite_id_seq")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```
Comment (saffchen) [inline stock/src/main/java/saffchen/entities/SatelliteEntity.java:18]: Why was the previous generation version changed

File: stock/src/main/java/saffchen/entities/PersonEntity.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import javax.persistence.*;
+
+@Entity
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode
+@Table(name = "Person")
+public class PersonEntity {
```
Comment (saffchen) [inline stock/src/main/java/saffchen/entities/PersonEntity.java:14]: Do we can use  other class name? Title "PersonEntity" not clear.

File: stock/src/main/java/saffchen/entities/PersonEntity.java (hunk 1 above)
Comment (saffchen) [inline stock/src/main/java/saffchen/entities/PersonEntity.java:20]: If we don't need this annotation then delete it

**Review by saffchen — CHANGES_REQUESTED** [review]

File: stock/src/main/java/saffchen/entities/PersonEntity.java (hunk 3 above)
Comment (freesky15) [reply stock/src/main/java/saffchen/entities/PersonEntity.java:14]: Sorry, I can't understand what do you mean?

**Comment by freesky15** [comment]
The Branch was updated

**Pull request closed without merging**